modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -10,35 +10,31 @@
 [id="registry-configuring-storage-vsphere_{context}"]
 = Configuring registry storage for VMware vSphere
 
-As a cluster administrator, following installation you must configure your
-registry to use storage.
+As a cluster administrator, following installation you must configure your registry to use storage.
 
 .Prerequisites
 
 * Cluster administrator permissions.
 * A cluster on VMware vSphere.
-* Provision persistent storage for your cluster. To deploy a private image registry, your storage must provide
-ReadWriteMany access mode.
+* Persistent storage provisioned for your cluster.
 +
 [IMPORTANT]
 ====
-vSphere volumes do not support the `ReadWriteMany` access mode. You must use
-a different storage backend, such as object storage, to configure the registry storage for high-availability.
+{product-title} supports `ReadWriteOnce` access mode for image registry storage when you have only one replica. To deploy a private image registry, or to configure registry storage for high availability with two or more replicas, you must use a different storage back end that supports `ReadWriteMany`, such as object storage.
```

Comment:
Didn't if we tested object storage in vSphere either.

Reply:
We have a similar note already in 4.4, 4.5: 

> vSphere volumes do not support the ReadWriteMany access mode. You must use a different storage backend, **_such as object storage_**, to configure the registry storage for high-availability.

So is it accurate still to use the wording "such as object storage", or should I remove those words in 4.4+?